build/Versions.props

```diff
@@ -37,6 +38,7 @@
   <PropertyGroup>
     <MicrosoftNetCompilersVersion>2.4.0</MicrosoftNetCompilersVersion>
     <MicrosoftNetCoreCompilersVersion>2.6.0-beta3-62309-01</MicrosoftNetCoreCompilersVersion>
+    <MicrosoftCodeAnalysisBuildTasksVersion>3.0.0-beta1-61516-01</MicrosoftCodeAnalysisBuildTasksVersion>
```

Comment:
According to https://dotnet.myget.org/feed/roslyn/package/nuget/Microsoft.CodeAnalysis.Build.Tasks the 3.0.0-beta1-61516-01 version was pushed on Thu, 16 Mar 2017 while e.g. 2.6.0-beta3-62329-03 was pushed yesterday, is this the correct version to use?

Reply:
Doubt it. This entire branch needs to get reviewed. It just didn't build as-is so I wanted to get that in quick. Good catch though, will add it to the list to ensure they're in sync (if possible).